portal-api/src/v1/validation/route-validators/route-validators.js

```diff
@@ -1,8 +1,13 @@
-const { param } = require('express-validator');
+const { check } = require('express-validator');
 
-const bankIdValidation = param('bankId')
+const bankIdValidation = check('bankId')
+  .exists()
+  .withMessage('No bank id was provided')
   .isString()
+  .withMessage('The bank id provided should be a string of numbers')
```

Comment:
Does this not fall through to the bottom message which is the same here?

Reply:
It doesn't seem very clear but I think `withMessage` sets the message for the previous validation step only